include/erlang_ls.hrl

```diff
@@ -590,14 +590,16 @@
                    | module
                    | parse_transform
                    | record
-                   | record_access
+                   | record_def_field
                    | record_expr
+                   | record_field
                    | spec
                    | type_application
                    | type_definition
                    | variable.
 -type poi_range() :: #{ from := pos(), to := pos() }.
 -type poi_id()    :: atom()
+                   | {atom(), atom()} %% record_def_field, record_field
```

Comment:
(outside of the PR scope) We should probably create aliases for these ids (eg `poi_id_record_field()`).

Reply:
good idea